packages/web/src/components/result/ReactiveList.js

```diff
@@ -119,7 +127,7 @@ class ReactiveList extends Component {
 		this.setReact(this.props);
 
 		this.domNode = window;
-		if (!this.props.pagination) {
+		if (this.showInfiniteScroll) {
```

Comment:
Infinite scroll doesn't have an entity to render - so not sure if this prop name makes sense? I think we can remove `show` from here.